back/config/schemas/style.schema.json

```diff
@@ -52,7 +52,7 @@
       "$ref": "#/definitions/color"
     },
     "signedInHeaderOverlayOpacity": {
-      "description": "The opacity of the overlay of the signed-in header (0 (no color) to 100 (full color))",
+      "description": "The opacity of the overlay of the signed-in header (0 (no color) to 100 (full color)). Does not affect the signed-in header opacity when the signed-out header layout is fixed-ratio. In this case the signed-in header overlay defaults to 100 opacity (no image visible)",
```

Comment:
Is it the only change? Does the rest come from master? If yes, good to merge.

Reply:
Yes, the only change :+1: